### PR TITLE
#2771 - Fix - Textstring "and" connecting Terms of Services and Privacy Policy is hardcoded and cannot be translated 

### LINKED ISSUE #2771: Textstring "and" connecting Terms of Services and Privacy Policy is hardcoded and cannot be translated
In the bp-core/bp-core-catchuri.php file on line 1520 there is a hardcoded string "and" which therefore cannot be translated.

**Jira issue** : [PROD-768]

[PROD-768]: https://buddyboss.atlassian.net/browse/PROD-768?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### PR DESCRIPTION
**Pull request by meetmakadiya**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2771  .

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-core/bp-core-catchuri.php [hunk 1]
```diff
@@ -1522,7 +1522,7 @@ function bp_core_change_privacy_policy_link_on_private_network( $link, $privacy_
 			);
 			$page_title  = ( $privacy ) ? get_the_title( $privacy ) : '';
 			$get_privacy = get_post( $privacy );
-			$link        .= ' and ';
+			$link        .= esc_html__( ' and ','buddyboss' );
```
Comment (chetansatasiya) [inline src/bp-core/bp-core-catchuri.php:1525]:
```suggestion
			$link        .= esc_html__( ' and ', 'buddyboss' );
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need proper spacing

File: src/bp-core/bp-core-catchuri.php (hunk 1 above)
Comment (meetmakadiya) [reply src/bp-core/bp-core-catchuri.php]: @chetansatasiya updated with proper space.

**Review by dipskakadiya — APPROVED** [review]

**Comment by meetmakadiya** [comment]
Duplicate PR so I close this PR
https://github.com/buddyboss/buddyboss-platform/pull/2599

**Pull request closed without merging**